### PR TITLE
Added missing hungarian translations, fix typos

### LINKED ISSUE #22503: Add missing hungarian messages
### Description

Add missing hungarian translations, fix some typos

### PR DESCRIPTION
**Pull request by nkelemen18**
Resource files synchronized with the original English files. Added missing entries and fixed some Hungarian typos.

Closes: #22503

### CODE REVIEW

**Comment by ahus1** [comment]
Thank you for your contribution. Please someone from the Keycloak community to review your translations and leave a comment or review in this issue. Once there is a review this PR can be merged. Thanks!

File: themes/src/main/resources-community/theme/base/login/messages/messages_hu.properties [hunk 1]
```diff
@@ -14,9 +14,12 @@ doClickHere=Kattintson ide
 doImpersonate=Megszemélyesítés
 doTryAgain=Próbálja újra
 doTryAnotherWay=Pórbálja máshogyan
+doConfirmDelete=Törlése megerősítése
```
Comment (Markerer) [inline themes/src/main/resources-community/theme/base/login/messages/messages_hu.properties:17]:
```suggestion
doConfirmDelete=Törlés megerősítése
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_hu.properties [hunk 2]
```diff
@@ -165,53 +210,80 @@ client_broker=Ügynök
...
+federatedIdentityConfirmReauthenticateMessage=Azonosítsa magát, hogy összeköthesse a felhasználói fiókját a következővel: {0}
+nestedFirstBrokerFlowMessage=A(z) {0} {1} felhasználó nincs összekötve egyetlen ismert felhasználóval sem.
 confirmLinkIdpReviewProfile=Fiók áttekintése
 confirmLinkIdpContinue=Hozzáadás meglévő fiókhoz
 
 configureTotpMessage=Fiókja aktiválásához előbb be kell állítania egy mobil hitelesítő eszközt.
+configureBackupCodesMessage=Fiókja aktiválásához előbb létre kell hoznia Biztonsági Kódokat.
 updateProfileMessage=Fiókja aktiválásához előbb módosítania kell a felhasználói adatait.
 updatePasswordMessage=Fiókja aktiválásához előbb le kell cserélnie a jelszavát.
+updateEmailMessage=Fiókja aktiválásához előbb le módosítania kell e-mail címét.
```
Comment (Markerer) [inline themes/src/main/resources-community/theme/base/login/messages/messages_hu.properties:280]:
```suggestion
updateEmailMessage=Fiókja aktiválásához előbb módosítania kell e-mail címét.
```

File: themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_hu.properties [hunk 3]
```diff
@@ -1 +1,174 @@
...
+shareSuccess=Az erőforrás sikeresen megosztva
+unShareSuccess=Az erőforrás megosztása megszüntetve
+updateSuccess=Az erőforrás módosítva
+resourceAlreadyShared=Az erőforrás már meg van osztva ezzel a felhasználóval.
+resourceNotShared=Ez az erőforrás nincs megosztva
+permissionRequests=Jogosultság kérelmek
+permissions=Jogosultságok
+selectPermissions=Jogosultságok kiválasztva
+unShareAllConfirm=Valóban meg szeretné szüntetni az összes megosztást?
+userNotFound=Nem található felhasználó a(z) {0} névvél vagy E-mail címmel.
```
Comment (Markerer) [inline themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_hu.properties:69]:
```suggestion
userNotFound=Nem található felhasználó a(z) {0} névvél vagy e-mail címmel.
```

File: themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_hu.properties [hunk 4]
```diff
@@ -1 +1,174 @@
...
+webauthn-display-name=Biztonsági kulcs
+webauthn-help-text=Biztonsági kulcs használata bejelentkezéshez.
+webauthn-passwordless-display-name=Biztonsági kulcs
+webauthn-passwordless-help-text=Biztonsági kulcs használata jelszó nélküli bejelentkezéshez.
+basic-authentication=Normál bejelentkezés
+invalidRequestMessage=Érvénytelen kérés
+
+# Applications page
+applicationsPageTitle=Alkalmazások
+applicationsPageSubTitle=Alkalmazások jogosultságainak kézelése
```
Comment (Markerer) [inline themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_hu.properties:118]:
```suggestion
applicationsPageSubTitle=Alkalmazások jogosultságainak kezelése
```

File: themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_hu.properties [hunk 5]
```diff
@@ -1 +1,174 @@
...
+error-user-attribute-required=Kérem, adja meg a(z) ''{0}'' értékét.
+error-invalid-date=''{0}'' érvénytelen dátum.
+error-username-invalid-character=''{0}'' érvénytelen karaktert tartalmaz.
+error-person-name-invalid-character='{0}' érvénytelen karaktert tartalmaz.
+
+updateEmail=E-mail cím módosítása
+
+#groups
+groupLabel=Csoportok
+groupDescriptionLabel=Az Önhöz tartozó csoportok megtakintése
```
Comment (Markerer) [inline themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_hu.properties:170]:
```suggestion
groupDescriptionLabel=Az Önhöz tartozó csoportok megtekintése
```

**Review by Markerer — CHANGES_REQUESTED** [review]
Nice job, only some extra typo fixes are necessary.

**Review by Markerer — APPROVED** [review]

**Comment by nkelemen18** [comment]
> Nice job, only some extra typo fixes are necessary.

Thank you, your suggestions are applied.

@ahus1: I think, this PR is ready from our side.

**Review by ahus1 — DISMISSED** [review]
Thank you for updating the translation and the teamwork!

**Comment by nkelemen18** [comment]
@ahus1 I squashed my commits and rebased my branch on to of the main

**Comment by ahus1** [comment]
@nkelemen18 Please only rebase when you've been asked for once your changes have been approved, as rebases will discard previous approvals. I would have squashed and rebased the changes automatically.

**Review by ahus1 — APPROVED** [review]

**Pull request merged**